vendor/inline-scripts/pageview-tracking-experiment.js

```diff
@@ -0,0 +1,104 @@
+(function pageViewTracking(config = {}) {
+  let beacon;
+  let sessionId;
+  let pvNumber;
+  let pvNumberGlobal;
+  let headData;
+
+  function readCookies() {
+    beacon = getCookieValue('wikia_beacon_id');
+    sessionId = getCookieValue('tracking_session_id') || genUID();
+    pvNumber = getCookieValue('pv_number');
+    pvNumberGlobal = getCookieValue('pv_number_global');
+  }
+
+  function getCookieValue(cookieName) {
+    const cookieSplit = ('; ' + document.cookie).split('; ' + cookieName + '=');
+
+    return cookieSplit.length === 2 ? cookieSplit.pop().split(';').shift() : null;
+  }
+
+  function genUID() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+      const r = Math.random() * 16 | 0, v = c === 'x' ? r : (r & 0x3 | 0x8);
+
+      return v.toString(16);
+    });
+  }
+
+  function getEventParams(shouldIncreasePvNumber) {
+    let pvNumberComputed = pvNumber ? parseInt(pvNumber, 10) : 0;
+    let pvNumberGlobalComputed = pvNumberGlobal ? parseInt(pvNumberGlobal, 10) : 0;
+    if (shouldIncreasePvNumber) {
+      pvNumberComputed += 1;
+      pvNumberGlobalComputed += 1;
+    }
+
+    const params = {
+      beacon: beacon,
+      cb: new Date().valueOf(),
+      session_id: sessionId,
+      pv_unique_id: genUID(),
+      pv_number: pvNumberComputed,
+      pv_number_global: pvNumberGlobalComputed,
+      url: window.location.href,
+      c: headData.wikiVariables.id,
+      a: headData.trackingDimensions[21],
+      lc: headData.wikiVariables.language.content,
+      x: headData.wikiVariables.dbName,
+      s: 'mercury',
+      n: headData.trackingDimensions[25],
```

Comment:
namespace number took from tracking dimensions - that was the easiest way to get it 🤷‍♂️